src/parser.ts

```diff
@@ -570,6 +572,11 @@ function stateMachineStatementParser (statement: Statement, steps: Step[], { isS
         return;
       }
 
+      if (token.type === 'parameter') {
+        statement.parameters ||= []
```

Comment:
Just want to confirm that typescript compiles this down to something that isn't logical or as that's not supported on Node < 15, though again, totally fine (and would prefer) just making it part of `INITIAL_STATEMENT` and always be there for simplicity.